packages/client/components/MeetingSubnavItem.tsx

```diff
@@ -98,7 +111,18 @@ const MeetingSubnavItem = (props: Props) => {
       isUnsyncedFacilitatorStage={isUnsyncedFacilitatorStage}
       onClick={!isDisabled ? onClick : undefined}
     >
-      <ItemLabel isComplete={isComplete}>{children}</ItemLabel>
+      <Tooltip open={openTooltip}>
+        <TooltipTrigger asChild>
+          <ItemLabel
+            isComplete={isComplete}
+            onMouseEnter={handleMouseEnter}
```

Comment:
-1 the radix tooltip automatically shows the tooltip when you hover over the item, so adding `onMouseEnter` will conflict with this behaviour